simple_classifier/Classifiers/Display.py

```diff
@@ -0,0 +1,15 @@
+from simple_classifier.Classifiers.ClassifierProfiler import ClassifierProfiler
```

Comment:
```suggestion

```

Removed unused import. Display will take data as method parameters, no need to know anything about `ClassifierProfiler`.